src/quo2/components/community/banner/style.cljs

```diff
@@ -1,22 +1,21 @@
 (ns quo2.components.community.banner.style
-  (:require [quo2.foundations.colors :as colors]))
+  (:require [quo2.foundations.colors :as colors]
+            [quo2.foundations.shadows :as shadows]
+            [react-native.platform :as platform]))
 
 (defn community-card
   [theme]
-  {:shadow-offset     {:width 0 :height 2}
-   :shadow-radius     16
-   :shadow-opacity    1
-   :shadow-color      colors/shadow
-   :elevation         1
-   :border-radius     16
-   :justify-content   :space-between
-   :background-color  (colors/theme-colors colors/white colors/neutral-90 theme)
-   :flex-direction    :row
-   :margin-horizontal 20
-   :margin-top        8
-   :margin-bottom     8
-   :height            56
-   :padding-right     12})
+  (merge
+   (shadows/get (if platform/ios? 2 4) theme)
```

Comment:
you can use `assoc` as it's variadic. `merge` implies generally that you don't know the keys of the second parameter.

```
(assoc
  (shadows/get (if....)
   :border-radius 16
   :justify-content :space....
```

If you were to swap the parameters for example, if you would want to use `merge`

```
(merge
   {:border-radius 16
    ....}
   (shadows/get (if...)))
```

But  `assoc` should be used when keys are known